.github/workflows/build-wheels.yml

```
@@ -7,35 +7,176 @@ on:
             arch: x86_64
-            cibw_arch: x86_64
+            target: ""
+            manylinux: 2_28
+          # aarch64 wheels are built via QEMU emulation under maturin-action.
           - os: ubuntu-latest
             arch: aarch64
-            cibw_arch: aarch64
-          - os: windows-latest
+            target: aarch64-unknown-linux-gnu
+            manylinux: 2_28
+          - os: macos-13
```

actionlint (1.7.10)

label "macos-13" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)